From 829d0440eb9b44053a69fcd8f0ba683186480845 Mon Sep 17 00:00:00 2001
From: Jami Kettunen <jami.kettunen@protonmail.com>
Date: Mon, 9 Feb 2026 16:23:39 +0200
Subject: [PATCH] [HACK] minigbm_cros_gralloc_defaults: drop libhbm_minigbm dep

The prebuilts this relies on are unavailable for arm64; this most likely
breaks mesa but at least the image builds...
---
 Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.bp b/Android.bp
index 73aa46d..05ffeae 100644
--- a/Android.bp
+++ b/Android.bp
@@ -111,7 +111,7 @@ cc_defaults {
         "minigbm_headers",
     ],
 
-    static_libs: ["libarect", "libhbm_minigbm"],
+    static_libs: ["libarect"],
 
     vendor: true,
 
-- 
2.52.0

